libr/bin/bin.c

```diff
@@ -31,6 +31,13 @@ R_LIB_VERSION (r_bin);
 
 #define ARCHS_KEY "archs"
 
+#if !defined(R_BIN_STATIC_PLUGINS)
```

Comment:
I did call ./configure-plugins. In Debian I currently disabled the mach plugins (because of license, see #5868). I did this by removing the following entries from plugins.def.cfg and building normally via ./configure && make ([buildlog, with fix applied](https://buildd.debian.org/status/fetch.php?pkg=radare2&arch=i386&ver=0.10.6%2Bdfsg-1&stamp=1474961946)):

- bin.mach0
- bin.mach064
- bin.dyldcache
- bin_xtr.xtr_dyldcache
- bin_xtr.fatmach0

Without this I got an error in libr/bin/bin.c, that R_BIN_XTR_STATIC_PLUGINS is not defined. 
./configure-plugins does not generate it, since it only knows about BIN_XTR by plugins having a "bin_xtr." prefix. For R_BIN_STATIC_PLUGINS I just assumed, that the same problem exists, since I did not disable all bin plugins I did not run into this.

-- Sebastian